test/clt-tests/sharding/syntax/sample.rec

```diff
@@ -0,0 +1,10 @@
+––– block: ../base/init –––
+––– block: ../base/start-searchd-1 –––
+––– input –––
+mysql -h0 -P19306 -e "SELECT 'Hello world'"
```

Comment:
Do we really need this test here?

Reply:
Same about other sample.rec.

Reply:
I asked a question in the discussion, since there are similar files in other folders and, as I understand it, they are a sample for building other tests.

Reply:
Dmitrii has confirmed you can remove the sample.rec tests.

Reply:
Done